{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/templates/base.html

```diff
@@ -1,3 +1,5 @@
+{% import 'components/macros/components.html' as components %}
```

Comment:
This should not be in the `components` app; it's site-wide and on every site so it shouldn't be in an app folder.

Reply:
So should the macro go in `apps/site/templates/site/macros/site.html`?